webapp/src/modules/address/utils.js

```diff
@@ -1,3 +1,19 @@
 export function toAddressParcelIds(parcelsArray) {
   return parcelsArray.map(parcel => parcel.id)
 }
+
+export function toAddressPublicationIds(publicationsArray) {
+  return publicationsArray.map(publication => publication.tx_hash)
+}
+
+export function grab(all, ids) {
```

Comment:
Can we search for a name a bit more descriptive here? I think it's a bit generic and it prompts you to search the implementation to see what it does.

It's a bit hard to name because it's doing two things at the same time, but maybe:

`pickAndMap()`

ugh